ta/crypt/include/user_ta_header_defines.h

```diff
@@ -33,7 +33,7 @@
 #define TA_UUID TA_CRYPT_UUID
 
 #define TA_FLAGS		(TA_FLAG_USER_MODE | TA_FLAG_EXEC_DDR)
-#define TA_STACK_SIZE		(2 * 1024)
+#define TA_STACK_SIZE		(32 * 1024)
```

Comment:
Why? Needed for mbedtls? Then, should be in the first commit that uses mbedtls I think.

Reply:
Yes, this is needed for mbedtls. Usually we try to do such things in separate commits.

Reply:
> Usually we try to do such things in separate commits.

What is the benefit? The drawback I see is it's harder to tell why we had to increase the size. It it's part of a commit that calls mbedtls then it's obvious why.

I'm be happy if you just mention the reason in the commit log however.